test/unit/routes/audit/test-audit.js

```diff
@@ -0,0 +1,90 @@
+const routeHelper = require('../../../helpers/routes/route-helper')
+const supertest = require('supertest')
+const expect = require('chai').expect
+const proxyquire = require('proxyquire')
+const sinon = require('sinon')
+
+let hasRolesStub
+let authorisation
+let getAuditDataStub
+
+describe('routes/audit', function () {
+  let app
+
+  beforeEach(function () {
+    hasRolesStub = sinon.stub()
+    authorisation = {
+      hasRoles: hasRolesStub
+    }
+    getAuditDataStub = sinon.stub().resolves([])
+
+    const route = proxyquire('../../../../app/routes/audit/audit', {
+      '../../services/authorisation': authorisation,
+      '../../services/data/audit/get-audit-data': getAuditDataStub
+    })
+
+    app = routeHelper.buildApp(route)
+  })
+
+  describe('GET /audit', function () {
+    it('should respond with a 200 when no audit data found', function () {
+      return supertest(app)
+        .get('/audit')
+        .expect(200)
+        .expect(function () {
+          expect(hasRolesStub.calledOnce).to.be.true //eslint-disable-line
```

Comment:
For this - and the other tests- it would be good to check what the dependencies are being called with rather than just checking they were called. E.g. Were the expected correct parameters sent?

Also, a lot of `// eslint-disable-line` everywhere. Might be worth figuring out why it's failing linting rules?

Reply:
> For this - and the other tests- it would be good to check what the dependencies are being called with rather than just checking they were called. E.g. Were the expected correct parameters sent?
> 
> Also, a lot of `// eslint-disable-line` everywhere. Might be worth figuring out why it's failing linting rules?

Yes planning to extend tests in next PR. Regarding eslint, I think they are everywhere in existing tests so can either disable globally on each tests or will see if they can be fixed.